packages/helpers/logger.ts

```diff
@@ -3,18 +3,25 @@ export interface Logger {
    success: (message: string) => void;
    warning: (message: string) => void;
    error: (message: string) => void;
+   timing: (statName: string, ms: number) => void;
+   event: (statName: string) => void;
 }
 
 export const log: Logger = {
    info: (message) => console.log(message),
    success: (message) => console.log(`\x1b[32m${message}\x1b[0m`),
    warning: (message) => console.log(`\x1b[33m${message}\x1b[0m`),
    error: (message) => console.log(`\x1b[31m${message}\x1b[0m`),
+   timing: (statName: string, ms: number) =>
+      console.log(`${statName} => ${ms.toFixed(2)}ms`),
+   event: (eventName: string) => console.log(`event ${eventName} -> 1`),
```

Comment:
I like this. One note is that I find colors helpful when reading Vercel logs.

Maybe we can define something more specific for datadog (that would also make it more obvious why we're formatting logs in a certain way)

```
log.datadog.event.success("hit");
log.datadog.timing("product", 100);
```

Thoughts? 🤔

Reply:
I thought about doing something like that, but decided I'd rather not have 6 functions (1 per color per type). We could take a color parameter, or a helper function that wraps a string in the escape codes that render color.

I'd rather not expose an implementation detail (the name of our current log aggregator) in the exported functions. I'll think about it a little.